### PR TITLE
[FrameworkBundle] [Workflow] Fix service marking store configuration

### PR DESCRIPTION
**Pull request by fduch**
| Q             | A
| ------------- | ---
| Branch?       | "master"
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | no
| License       | MIT
| Doc PR        | no

Currently workflow marking store configuration [checks](https://github.com/symfony/symfony/blob/3.2/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L271-L272) that `service` and `arguments` fields are not specified simultaneously using `isset` function.

Since [arguments node](https://github.com/symfony/symfony/blob/3.2/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L253-L261) defines prototype node inside, even if `arguments` node is not specified, -  after processing it looks like empty array. If `service` setting is set it leads to failing build with validation error message (due to `isset([])` returns `true`):
`"arguments" and "service" cannot be used together.`.

This patch addresses this issue.

### CODE REVIEW

**Comment by fduch** [comment]
Also tests added

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/workflows.yml [hunk 1]
```diff
@@ -63,3 +63,17 @@ framework:
...
+                - Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest
+            places:
+                - first
+                - last
+            transitions:
+                go:
+                    from:
+                        - first
+                    to:
+                        - last
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/workflows.yml]: please add the final EOL

**Review by stof — APPROVED** [review]

**Comment by stof** [comment]
:+1: 
Status: reviewed

**Comment by nicolas-grekas** [comment]
Thank you @fduch.

**Pull request closed without merging**